src/Profiles/Fec/ANTPLUS_FecDataPages.h

```diff
@@ -2,5 +2,14 @@
 #define ANTPLUS_PROFILEFECDATAPAGES_h
 
 #include <Profiles/Fec/DataPages/ANTPLUS_FecSpecificTrainerData.h>
+#include <Profiles/Fec/DataPages/ANTPLUS_FecManufacturerInformation.h>
+#include <Profiles/Fec/DataPages/ANTPLUS_FecProductInformation.h>

```

Comment:
According to the spec, manufacturers and product information are common datapages, we should use the existing types for that in the common datapages area